src/bin/inx-chronicle/migrations/migrate_1_0_0_beta_31.rs

```diff
@@ -0,0 +1,120 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use chronicle::{
+    db::{collections::OutputCollection, MongoDb, MongoDbCollection, MongoDbCollectionExt},
+    types::stardust::block::output::{AliasId, NftId, OutputId},
+};
+use futures::TryStreamExt;
+use mongodb::{bson::doc, options::IndexOptions, IndexModel};
+use serde::Deserialize;
+
+pub const PREV_VERSION: &str = "1.0.0-beta.30";
+
+pub async fn migrate(db: &MongoDb) -> eyre::Result<()> {
+    let collection = db.collection::<OutputCollection>();
+
+    #[derive(Deserialize)]
+    struct Res {
+        output_id: OutputId,
+    }
+
+    // Convert the outputs with implicit IDs
+    let outputs = collection
+        .aggregate::<Res>(
+            [
+                doc! { "$match": { "$or": [
+                    { "output.alias_id": AliasId::implicit() },
+                    { "output.nft_id": NftId::implicit() }
+                ] } },
+                doc! { "$project": {
+                    "output_id": "$_id"
+                } },
+            ],
+            None,
+        )
+        .await?
+        .map_ok(|res| res.output_id)
+        .try_collect::<Vec<_>>()
```

Comment:
We probably shouldn't collect those into memory but rather stream them, I guess.

Reply:
I agree that would be the correct way of doing things but I believe we will get rid of these migrations for a final RC anyways. So I would not worry about it now.